graphics2d/src/test/java/de/rototor/pdfbox/graphics2d/MultiPageTest.java

```diff
@@ -43,13 +46,18 @@ public void testMultiPageJFreeChart() throws IOException {
 		parentDir.mkdirs();
 		File targetPDF = new File(parentDir, "multipage.pdf");
 		PDDocument document = new PDDocument();
+        HashMap<Image, PDImageXObject> seenImages = new HashMap();
```

Comment:
No idea what IDE you are using, but you should get warnings from your IDE here because of not using generics correctly. I.e. that line should read

`
Map<Image, PDImageXObject> seenImages = new HashMap<Image, PDImageXObject>();
`